core/server/worker/src/test/java/alluxio/worker/netty/UfsFileWriteHandlerTest.java

```diff
@@ -50,7 +50,8 @@ public void before() throws Exception {
     UfsInfo ufsInfo = new UfsInfo(Suppliers.ofInstance(mockUfs), AlluxioURI.EMPTY_URI);
     Mockito.when(ufsManager.get(TEST_MOUNT_ID)).thenReturn(ufsInfo);
     Mockito.when(mockUfs.create(Mockito.anyString(), Mockito.any(CreateOptions.class)))
-        .thenReturn(mOutputStream);
+        .thenReturn(mOutputStream)
+        .thenReturn(new FileOutputStream(mFile, true));
```

Comment:
This seems a bit sketchy, should we just update this (or mock separately) in the individual test which needs it?

Reply:
I tried a few but don't see an obvious better way to mock this separately just for a test.